### PR TITLE
feat: Migrate from SSORealm to idp resources

### PR DESCRIPTION
**Pull request by Rolika4**
# Pull Request Template

## Description
Please include a summary of the change and why it is needed.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Pull Request contain one commit. I squash my commits

### CODE REVIEW

File: deploy-templates/templates/external-secrets/externalsecret-keycloak-broker.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: keycloak-client-broker-secret
+spec:
+  target:
+    template:
+      metadata:
+        labels:
+          app.edp.epam.com/secret-type: keycloak
```
[the remark this answers is not in the document]
Comment (Rolika4) [reply deploy-templates/templates/external-secrets/externalsecret-keycloak-broker.yaml]: removed

File: deploy-templates/templates/keycloak/keycloakauthflow.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  builtIn: false
+  authenticationExecutions:
+    - authenticator: "auth-cookie"
+      priority: 0
+      requirement: "ALTERNATIVE"
+    - authenticator: "identity-provider-redirector"
+      priority: 1
+      requirement: "ALTERNATIVE"
+      authenticatorConfig:
+        alias: EPAM-SSO
```
Comment (NikolayMarusenko) [inline deploy-templates/templates/keycloak/keycloakauthflow.yaml:23]: please change

Comment (Rolika4) [reply deploy-templates/templates/keycloak/keycloakauthflow.yaml]: done

**Review by NikolayMarusenko — APPROVED** [review]

**Pull request merged**